include/rosdiscover-clang/Value/Value.h

```diff
@@ -35,6 +36,9 @@ class SymbolicValue {
       return SymbolicValueType::Bool;
     } else if (typeName == "int") {
       return SymbolicValueType::Integer;
+    } else if (typeName == "float" 
```

Comment:
maybe add support for other integer types?

Reply:
what do you mean with other integer types? Like long?